internal/api/docs/openapi.yaml

```
@@ -1336,8 +1340,21 @@ components:
         variables:
           additionalProperties:
             type: string
+          description: 'Deprecated: use config_variables instead. This field is kept
+            for backward compatibility.'
           type: object
       type: object
+    BrickInstanceVariable:
```

should we change this?
```suggestion
    BrickConfigVariable:
```